src/selectors/producers.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'underscore';
+import { createSelector } from 'reselect';
+
+function selectSuppliersWithProducer(suppliers, producers) {
+  const entities = _.map(suppliers, (supplier) => {
+    const producer = _.omit(producers, ['created_at', 'updated_at', 'id']);
+
+    return {
+      ...producer,
+      ...supplier,
+    };
```

Comment:
How are you matching the right `supplier` with their `producer`. In the members selector I `findById` in users list